lua/sps/cl_init.lua

```diff
@@ -15,14 +15,54 @@ end)
 
 concommand.Add("ps_togglemenu", function(ply, cmd, args) PS_ToggleMenu() end)
 
+CreateClientConVar("ps_darkmode", "1", true)
+function SetPointshopTheme(dark)
+	PS_DarkMode = dark
+	RunConsoleCommand("ps_toggletheme")
+end
+SetPointshopTheme(!GetConVar("ps_darkmode"):GetBool())
```

Comment:
This won't work because you do RunConsoleCommand("ps_toggletheme") before you do concommand.Add("ps_toggletheme"...)

What I meant was, use the SetPointshopTheme function to set all the global variables like PS_TileBGColor so you can remove their other definitions (which have the same values copy pasted). Then initially call it with ps_darkmode and add a change callback to that convar to call it again when necessary.